packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -133,13 +167,53 @@ describe('Google Cloud Trace Exporter', () => {
       });
 
       assert.deepStrictEqual(
-        batchWrite.getCall(0).args[0].resource.spans[0].displayName.value,
+        batchWrite.getCall(0).args[0].spans[0].displayName.value,
         'my-span'
       );
-
+      assert.strictEqual(traceServiceConstructor.getCalls().length, 1);
+      assert.strictEqual(
+        traceServiceConstructor.getCall(0).args[0],
+        'cloudtrace.googleapis.com:443'
+      );
```

Comment:
Could we also test whether the second argument is what we would expect? Looking at the function body, I'm not sure how `grpc.credentials.createSsl()` or `grpc.credentials.createFromGoogleCredential(creds)` work. It seems like the latter uses the credentials returned from _getClient. So on line 92, maybe we could return some mock creds, and test if the output is what we would expect based on that mock. If the two `grpc.credentials` functions are non-deterministic somehow, then maybe it's more complicated, but let me know your thoughts.

Reply:
In the grpc-node, the [credentials test](https://github.com/grpc/grpc-node/blob/ea36f751a1874ed119556fb6782c10bbaafa64b8/packages/grpc-native-core/test/credentials_test.js#L401-L415) uses an in-process rpc server to verify that google credentials worked. There doesn't appear to be much of an interface to validate internally that the credentials are good.

I was thinking that I could spy the ssl and google credentials to verify that they are being called with the appropriate parameters (none, and `auth.getClient()`)?